README.md

```diff
@@ -1,31 +1,53 @@
-# grafana-opentelemetry-java
+<p>
+  <img src="https://upload.wikimedia.org/wikipedia/commons/3/3b/Grafana_icon.svg" alt="Grafana logo" height="70"/ >
+  <img src="https://opentelemetry.io/img/logos/opentelemetry-logo-nav.png" alt="OpenTelemetry logo" width="70"/ >
+</p>
+
+# Grafana OpenTelemetry distribution for Java
+
+[![Build](https://github.com/grafana/grafana-opentelemetry-java/actions/workflows/build.yml/badge.svg?branch=main)](https://github.com/grafana/grafana-opentelemetry-java/actions/workflows/build.yml)
+
+* [About](#about)
+* [Getting Started](#getting-started)
+* [Installation](#getting-started)
+* [Troubleshooting](#troubleshooting)
```

Comment:
In case you just added it for consistency, I removed the TOC for the .NET README (as I split up the README in several parts).

However, I guess it's fine here as this README is still longer.

Reply:
Just noticed that this is a build-in feature of github :smile: 

![image](https://github.com/grafana/grafana-opentelemetry-java/assets/2832627/ffc3f577-77e6-47de-a833-5474ddebd382)